services/signal_sciences.md

```diff
@@ -0,0 +1,25 @@
+# Signal Sciences
+
+We use Signal Sciences to monitor and limit traffic to the load balancer.
+
+## Access the UI
+
+[Signal Sciences load balancer overview](https://dashboard.signalsciences.net/corps/pu_library/sites/adc?dashboardId=5d93700e24dfc501917be84c)
+[View events](https://dashboard.signalsciences.net/corps/pu_library/sites/adc/events)
+[Enabled rules](https://dashboard.signalsciences.net/corps/pu_library/sites/adc/rules?enabled=true)
+
+## Rate limit and Bot traffic
+Sometimes a site gets hit with a lot of traffic from a bot that isn't respecting
+our robots.txt files. In this case you will likely see a big jump in requests to
+the machine (i.e. in Datadog) and you will see their user agent repeatedly in
+logs.
+
+We have an active rule to disallow traffic from any IP that makes > 200 requests
+in 1 minute. The traffic is disallowed for 10 minutes and then allowed again.
+The number of requests here was set just high enough to catch a specific bad bot we oserved on DPUL.
+
+If a bot is slowing down a service despite that rate limit, you can make a rule
```

Comment:
```suggestion
If a bot is slowing down a service without triggering the automatic ban, you can make a rule
```

Reply:
Honestly sometimes it seems like the ban is triggered but it still doesn't fix the traffic issues.